temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -273,6 +292,83 @@ public Builder setSslContext(SslContext sslContext) {
       return this;
     }
 
+    /**
+     * Sets default SSL context parameters that can be used with the temporal cloud service. Users
+     * that require additional customization may use {@link #setSslContext(SslContext)} directly.
+     *
+     * @param keyCertChainInputStream - an input stream for an X.509 certificate chain in PEM
+     *     format.
+     * @param keyPassword - the password of the keyFile, or null if it's not password-protected.
+     * @param keyInputStream - an input stream for a PKCS#8 private key in PEM format.
+     * @param fingerprints - a list of SHA1 fingerprints in hexadecimal form.
+     * @throws SSLException - when it was unable to build the context.
+     */
+    public Builder setCloudSslContext(
```

Comment:
might be worth having a non-default helper option where the user doesn't care about doing TLS server validation (such as for development purposes). We have the same thing for the GoSDK (InsecureSkipVerify) that has been really useful.

Reply:
There is a discussion point about this, which we should address. Namely what should be the default behavior in case if fingerprints are not set. This could default to Insecure or our default root CA fingerprint. I think if we hardcode our root CA fingerprint then using insecure will not be required at all. In other words I don't want to create wrong incentives and provide an easy way of using insecure as it may eventually leak into production code.

Reply:
On the second thought, let me add an insecure method, and just call it insecure explicitly. I think this would be a better user experience so worth doing.